apps/ngrx/effect-selector/README.md

```diff
@@ -11,3 +11,13 @@ npx nx serve ngrx-effect-selector
 ### Documentation and Instruction
 
 Challenge documentation is [here](https://angular-challenges.vercel.app/challenges/ngrx/2-effect-selector/).
+
+## Useful Resources
+
+- [Github](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/docs/rules/avoid-combining-selectors.md)
+- [Blog](https://brandonroberts.dev/blog/posts/2020-12-14-maximizing-simplifying-component-views-ngrx-selectors#building-view-models)
+- [Dev.to](https://dev.to/this-is-angular/ngrx-tips-i-needed-in-the-beginning-4hno)
+- [YouTube](https://www.youtube.com/watch?v=aTrD0VOBFso)
+- [Github](https://github.com/ngrx/platform/issues/3719)
+- [NgRx](https://ngrx.io/guide/store/feature-creators)
+- [In Depth Dev](https://indepth.dev/posts/1442/ngrx-bad-practices)
```

Comment:
if you add them inside another PR, we can merge them, That can always help someone

Reply:
It would be helpful but that is part of the challenge.  As a developer, you always have to look things up and find examples of code that can give you ideas how to fix your problem.